### ISSUE TITLE
Steamdeck aspect ratio 16:10 issue, TV-video screen not filled completely

### ISSUE THREAD

**Issue by Derik-for-real24** [CkauNui/ckau-book#113]
First of all Big thanks to the recent update fixing black borders in some vieuwstyle modes, its a bless.

However, one very small issue that remains only affects the Steamdeck, which I suspect is because of its aspect ratio 16:10 at resolution 1280 x 800. You see, when your navigating in the main menu between your platforms where it shows tv screen of platforms playing videos, interestingly, for some platforms the tv screen is not filled entirely when video plays, see picture below.

![IMG_20231025_194236](https://github.com/CkauNui/ckau-book/assets/108466376/5743585b-cba1-4e54-83bc-4d61edf334d8)

Based on the picture above, you can clearly see the tv screen is not getting filled completely, as such this issue only effects the steamdeck running with 16:10 aspect ratio. But on my main PC running 4k 16:9, this issue does not exist at all, tv screen platforms are all completely filled. 

One small point about the steamdeck, TV screen platforms such as (**Game Cube, GBA, Megadrive, dreamcast, psx**) did not suffer from screen not getting filled when video plays, videos for these platforms I mentioned above, specifically where scraped from **ScreenScraper** nd they filled the screen completely on the steamdeck which is interresting. 

But for other TV screen platforms such as **Windows, PS3, Wii, Switch** , I basically rip the videos from youtube for these platforms as mp4 file nd they differ from 720 to 1080p vids, but unfortunately these video suffer from tv screen not getting filled entirely when video plays, and its only happening on the steamdeck (16:9), not on main pc running 4k 16:9.

In short, my analogy of the issue, is that steamdeck aspect ratio 16:10 causes this issue to happen since my main pc (16:9) does not have that problem at all. This issue is only about the navigation screen where the user scrolls through his platforms in the form of a tv screen.
